### PR TITLE
[zh-cn]: create the translation of `tabs.MutedInfoReason` type

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+指定标签页静音或取消静音的原因。
+
+## 类型
+
+此类型的值是字符串，表示标签页被静音或取消静音的原因。可能的取值包括：
+
+- “capture”
+  - : 开始标签页捕获，强制更改静音状态。
+- “extension”
+  - : 扩展程序设置了静音状态。如果原因是扩展程序设置的，{{WebExtAPIRef("tabs.mutedInfo")}} 中的 `extensionId` 属性将包含负责的扩展程序的 ID。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:19]:
```suggestion
  - : 扩展程序设置了静音状态。如果是这个原因，{{WebExtAPIRef("tabs.mutedInfo")}} 中的 `extensionId` 属性将包含负责的扩展程序的 ID。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+{{AddonSidebar}}
+
+指定标签页静音或取消静音的原因。
+
+## 类型
+
+此类型的值是字符串，表示标签页被静音或取消静音的原因。可能的取值包括：
+
+- “capture”
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:16]: 和其他的 PR 一样，这里就用半角引号吧

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - : 开始标签页捕获，强制更改静音状态。
+- “extension”
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:18]: +1

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - : 扩展程序设置了静音状态。如果原因是扩展程序设置的，{{WebExtAPIRef("tabs.mutedInfo")}} 中的 `extensionId` 属性将包含负责的扩展程序的 ID。
+- “user”
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:20]: +1

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md (hunk 2 above)
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:16]:
```suggestion
- "capture"
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md (hunk 3 above)
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:18]:
```suggestion
- "extension"
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md (hunk 4 above)
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/tabs/mutedinforeason/index.md:20]:
```suggestion
- "user"
```

**Review by yin1999 — COMMENTED** [review]
:+1:

**Pull request merged**